src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -1154,3 +1154,71 @@ def restore_storage_account(client, vault_base_url, file_path):
         data = file_in.read()
         return client.restore_storage_account(vault_base_url, data)
 # endregion
+
+
+# region private_endpoint
+def show_private_endpoint_connection(client, resource_group_name, vault_name, connection_name,
+                                     connection_id=None):  # pylint: disable=unused-argument
+    """Show details of a private endpoint connection associated with a Key Vault."""
+    return client.get(resource_group_name=resource_group_name, vault_name=vault_name,
+                      private_endpoint_connection_name=connection_name)
```

Comment:
If this is a 1:1 mapping, maybe we can eliminate the custom function?

Reply:
@jiasli The only thing I have to do here is using name `connection_name` instead of `private_endpoint_connection_name`, because of this minor mismatch, I can not directly map it to SDK method, do you have any suggestion?

Reply:
Let's use `extra` to attach parameters. https://github.com/Azure/azure-cli/blob/cc53d1f95e3956444b885db0f3f48c2076cd3388/src/azure-cli/azure/cli/command_modules/keyvault/_params.py#L148